Generator/Stubs/model.stub

```diff
@@ -4,9 +4,23 @@ declare(strict_types=1);
 
 namespace App\Containers\{{section-name}}\{{container-name}}\Models;
 
-use App\Ship\Parents\Models\Model;
-
-class {{class-name}} extends Model
+use App\Ship\Parents\Models\Model as ParentModel;
+use OpenApi\Attributes\Property;
+use OpenApi\Attributes\Schema;
+use OpenApi\Attributes\Tag;
```

Comment:
It seems to me that we do not need to add to the stubs library which is not in the dependencies

Reply:
OK.